Build-JellyFin.ps1

```diff
@@ -8,6 +8,13 @@ param(
     [ValidateSet('win','win7', 'win8','win81','win10')][string]$WindowsVersion = 'win',
     [ValidateSet('x64','x86', 'arm', 'arm64')][string]$Architecture = 'x64'
 )
+
+if($IsWindows){
+    $TempDir = $env:Temp
+}else{
+    $TempDir = mktemp -d 
+}
+
 function Build-JellyFin {
     if($Architecture -eq 'arm64'){

```

Comment:
Any reason not to have an and condition on this if statement rather than nesting?